src/Banner/Banner.tsx

```diff
@@ -1,22 +1,22 @@
 import './Banner.scss'
-import { CurrentPark, Name, CurrentParkContainer } from '../ParkContainer/interfaces'
+import { CurrentPark, Name, CurrentParkContainer, Images, ImageObject} from '../ParkContainer/interfaces'
 import BannerIcons from '../BannerIcons/BannerIcons'
 
-type BannerProps = CurrentPark | CurrentParkContainer | Name
+type BannerProps = CurrentPark | CurrentParkContainer | Name | Images | ImageObject
 
 const Banner: React.FC<BannerProps> = props => {
-
+console.log(props)
     const { currentPark } = props as CurrentParkContainer
-    // const { currentPark } = props as CurrentParkContainer
-    // const { fullName } = props as Name
-    // const { url, altText } = props as Images
+    const { images } = currentPark as any
```

Comment:
We may want to change this "any" type to an array of objects later, but couldn't figure that out with an interface on this branch. "any" works for now.